src/renderer/containers/UploadSelectionPage/UploadTypeSelector/index.tsx

```diff
@@ -19,6 +19,12 @@ export default function UploadTypeSelector() {
         dispatch(selectUploadType(e.target.value));
     }
 
+    // Help text below the radio group. Defaults to the text for UploadType.File
+    let additionalHelpText = `Select one or more standalone files. Attempting to select ${UploadType.Multifile}s or folders will result in an error.`
+    if (uploadType === UploadType.Multifile) {
```

Comment:
Isnt multifile getting renamed 'chunked file' by the higher-ups?

Reply:
Is it possible to disable folders from selection?

Reply:
> Isnt multifile getting renamed 'chunked file' by the higher-ups? AH, the enum is probably updated. Never mind.

Yeah, should get caught by the enum change.

 > Is it possible to disable folders from selection?

Yup! That's what it does now. [Electron makes it very easy](https://github.com/aics-int/aics-file-upload-app/blob/main/src/renderer/containers/UploadSelectionPage/index.tsx#L35-L45).